README.md

```diff
@@ -20,9 +20,20 @@ vulkan-zig aims to be always compatible with the ever-changing Zig master branch
 ### CLI-interface
 A CLI-interface is provided to generate vk.zig from the [Vulkan XML registry](https://github.com/KhronosGroup/Vulkan-Docs/blob/main/xml), which is built by default when invoking `zig build` in the project root. To generate vk.zig, simply invoke the program as follows:
 ```
-$ zig-out/bin/vulkan-zig-generator path/to/vk.xml output/path/to/vk.zig
+$ zig-out/bin/generator path/to/vk.xml output/path/to/vk.zig
 ```
 This reads the xml file, parses its contents, renders the Vulkan bindings, and formats file, before writing the result to the output path. While the intended usage of vulkan-zig is through direct generation from build.zig (see below), the CLI-interface can be used for one-off generation and vendoring the result.
+NOTE: you need to replace `path/to/vk.xml` with the spec path from whatever source you prefer, here are some example orderered from the most recommended:
```

Comment:
- The SDK version is preferred because it is stable, though that can also be obtained from git. Regardless, i suggest swapping the order.
- VULKAN_SDL is a spelling error
- Maybe its better to leave out the commands, and just write that the file can be found at share/vulkan/registry in the SDK?

Reply:
I wrote SDL too many times, gotta touch some grass xD
Yea I'll try to resolve the suggestions